app/models/invitation.rb

```diff
@@ -0,0 +1,47 @@
+class Invitation < ActiveRecord::Base
+  include AASM
+
+  belongs_to :task, inverse_of: :invitations
+  has_one :paper, through: :task
+  belongs_to :invitee, class_name: "User", inverse_of: :invitations
+  belongs_to :actor, class_name: "User", inverse_of: :invitations
+
+  after_commit :notify_invitation_invited, on: :create
+
+  aasm column: :state do
+    state(:invited, {
+      initial: true,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.